vdom/tests/test_core.py

```diff
@@ -79,7 +53,7 @@ def test_to_json():
         'div',
         'children': [{
             'tagName': 'h1',
-            'children': 'Our Incredibly Declarative Example',
+            'children': ['Our Incredibly Declarative Example'],
```

Comment:
It's funny looking at these now given how I test a lot of React code -- it makes me wish for jest snapshots in Python. Is there something similar?

Reply:
I don't know of anything like that, but I recently discovered https://hypothesis.readthedocs.io/en/latest/ which might help!